src/assets/third-party/prism.css

```diff
@@ -0,0 +1,141 @@
+/* PrismJS 1.15.0
```

Comment:
This file should not be necessary, you can pick it from the npm package https://www.npmjs.com/package/prismjs